### ISSUE TITLE
jsx-curly-spacing accepted formats

### ISSUE THREAD

**Issue by OoDeLally** [palantir/tslint-react#249]
### Bug Report

- __tslint-react version__: 4.1.0
- __tslint version__: 5.20.1
- __typescript version__: 3.7.4
- __Running TSLint via__: VSCode

#### TypeScript code being linted

```ts
<IonToast
        isOpen={ Boolean(toastMessage)}
        onDidDismiss={handleDismiss }
        buttons={[{
            text: 'Ok',
            role: 'cancel',
            handler: handleDismiss,
          }]}
/>
```

with `tslint.json` configuration:

```json
{
    "jsx-curly-spacing": [
      2,
      {
        "when": "always",
        "allowMultiline": true
      }
    ],
}
```

#### Actual behavior

The rule is ignored.

However

```json
{
    "jsx-curly-spacing": [
      true,
      "always"
    ],
}
```
works.

#### Expected behavior

Should behave the same way as https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-curly-spacing.md

**Comment by adidahiya** [comment]
This package is being deprecated, see https://github.com/palantir/tslint-react/issues/210.

Also, I recommend using prettier for code formatting.
